### PR TITLE
Completed Issue 1 create an entry - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-05-30 at 10 56 48 AM

### CODE REVIEW

**Comment by j-alejandro-araujo** [comment]
Screenshot 2023-05-30 at 1 37 59 PM

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Verdana, sans-serif;
```
Comment (shawnkost) [inline css/styles.css]: We don't use Verdana in this project. The two fonts we use are Open Sans, and Proza Libre

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+.head-background {
+  background-color: #562a81;
+  width: 100%;
+  height: 5rem;
+}
+
+#code-journal-header {
+  color: white;
+  font-family: "Gill Sans", sans-serif;
```
Comment (shawnkost) [inline css/styles.css]: Same note as above ^

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  padding-bottom: 1.5rem;
+}
+
+#new-entry {
+  margin: 0 1rem;
+}
+
+#entry-background {
+  min-height: 100vh;
+  max-width: 900px;
```
Comment (shawnkost) [inline css/styles.css:45]: Works, but we already have a `max-width` set on the `.container` class, so we can just utilize that class on an element to contain the `width`

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  padding: 0.5rem 1rem;
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+  margin-left: auto;
+}
+
+@media screen and (min-width: 768px) {
+  .column-half {
+    flex-basis: 50%;
+  }
```
Comment (shawnkost) [inline css/styles.css]: This is already happening in your `layout.css` file

File: index.html [hunk 5]
```diff
@@ -1,30 +1,56 @@
...
-  <header>
-    <div class="container">
+  <header></header>
+  <main>
+    <div class="head-background">
+      <div class="container">
+        <nav>
+          <p id="code-journal-header">Code Journal</p>
+        </nav>
+      </div>
```
Comment (shawnkost) [inline index.html:24]: This should all be inside your `header` on line 18

File: js/data.js [hunk 6]
```diff
@@ -6,3 +6,26 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var dataJSON = JSON.stringify(data);
```
Comment (shawnkost) [inline js/data.js]: Use `const`

File: js/data.js [hunk 7]
```diff
@@ -6,3 +6,26 @@ var data = {
...
+  localStorage.setItem('data', dataJSON);
+});
+
+if (localStorage.getItem('data')) {
+  var storedDataJSON = localStorage.getItem('data');
```
Comment (shawnkost) [inline js/data.js]: Use `const`

File: js/data.js [hunk 8]
```diff
@@ -6,3 +6,26 @@ var data = {
...
+// }
+
+// deleteLocalData();
+
+// data = {
+//   view: 'entry-form',
+//   entries: [],
+//   editing: null,
+//   nextEntryId: 1
+// };
```
Comment (shawnkost) [inline js/data.js]: Delete all old code

File: js/main.js [hunk 9]
```diff
@@ -0,0 +1,28 @@
+const $photoUrlInput = document.querySelector('#photo-url');
+const $photoPreview = document.querySelector('#image');
+
+$photoUrlInput.addEventListener('input', () => {
+  const $imageUrl = $photoUrlInput.value;
```
Comment (shawnkost) [inline js/main.js]: No need for the `$` here, we only need to include it when storing DOM elements

File: js/main.js [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+const $form = document.querySelector('#new-entry');
+const $titleInput = document.querySelector('#title');
+const $notesInput = document.querySelector('#notes');
+
+$form.addEventListener('submit', event => {
+  event.preventDefault();
+
+  var entry = {
```
Comment (shawnkost) [inline js/main.js]: Use `const`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Jesus! A few changes requested here

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**